site/static/main.css

```diff
@@ -260,6 +261,10 @@ nav a:hover {
   text-decoration: none;
 }
 
+.title h1 {
+  margin-top: 0.3em;
```

Comment:
I didn't like the gap in the title. That's why I added it there. Even Vega does it